src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Commands/PriorLearningDataHandlerTests.cs

```diff
@@ -130,63 +127,88 @@ public async Task Handle_WhenRecognisePriorLearningIsNotTrue_ExceptionIsThrown(b
             fixture.VerifyException<DomainException>();
         }
 
-        [Test]
-        public async Task Handle_WhenNoDurationReducedByRplIsSet_ExceptionIsThrown()
+        [TestCase(null, "Please select Yes or No")]
+        public async Task Handle_WhenNoDurationReducedByRplIsSet_ExceptionIsThrown(bool? cval, string error)
         {
             fixture = new PriorLearningDataHandlerTestsFixture();
-            fixture.Command.IsDurationReducedByRpl = null;
+            fixture.Command.IsDurationReducedByRpl = cval;
             await fixture.Handle();
 
             var exception = fixture.Exception as DomainException;
             var domainErrors = exception.DomainErrors.ToList();
 
             domainErrors.Count().Should().BeGreaterThan(0);
 
-            domainErrors.Any(e => e.PropertyName == "isDurationReducedByRpl" && e.ErrorMessage == "Please select Yes or No").Should().Be(true);
-            domainErrors.Any(e => e.PropertyName == "costBeforeRpl" && e.ErrorMessage == "RPL total reduced price should be less than the total price for the apprenticeship").Should().Be(true);
+            domainErrors
+                .Any(e => 
+                e.PropertyName == "isDurationReducedByRpl" &&
+                e.ErrorMessage == error).Should().Be(true);
 
         }
 
-        [Test]
-        public async Task Handle_WhenNoCostBeforeRplIsSet_ExceptionIsThrown()
+        [TestCase(null, "You must enter the price, the price can't be negative, the price must be 35000 or less")]
```

Comment:
Could we also use testsCases for 

TestCase(-1, "The hours can't be negative")
TestCase(100000, "The hours entered must be 35000 or less")